data-connect/README.md

```diff
@@ -0,0 +1,5 @@
+# Firebase Data Connect
+
+<a href="https://idx.google.com/new?template=https://github.com/project-idx/community-templates/tree/mtewani-flutter-template/dataconnect">
```

Comment:
Needs to be /templates and also not your branch

Reply:
Updated.